JavaDirectory/src/test/java/com/bug/free/invention/api/unitTest/US002Test.java

```diff
@@ -0,0 +1,39 @@
+package com.bug.free.invention.api.unitTest;
+
+import com.bug.free.invention.api.controllers.US002;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
```

Comment:
I didn't see JUnit 5 in the pom - be careful about where your libraries are coming from. (I think IntelliJ maybe provides these on the classpath by default?)